pymc3/tests/test_distributions_random.py

```diff
@@ -1306,6 +1303,56 @@ class TestOrderedProbit(BaseTestDistribution):
     ]
 
 
+class TestWishart(BaseTestDistribution):
+    def wishart_rng_fn(self, size, nu, V, rng):
+        return st.wishart.rvs(nu, V, size=size, random_state=rng)
+
+    pymc_dist = pm.Wishart
+
+    V = np.eye(3)
+    pymc_dist_params = {"nu": 4, "V": V}
+    reference_dist_params = {"nu": 4, "V": V}
+    expected_rv_op_params = {"nu": 4, "V": V}
+
+    reference_dist = lambda self: functools.partial(
+        self.wishart_rng_fn, rng=self.get_random_state()
+    )
+    tests_to_run = [
+        "check_rv_size",
+        "check_pymc_params_match_rv_op",
+        "check_pymc_draws_match_reference",
+    ]
+
+
+class TestMatrixNormal(BaseTestDistribution):
+    def matrixnormal_rng_fn(self, size, mu, rowcov, colcov, rng):
+        return st.matrix_normal.rvs(
+            mean=mu, rowcov=rowcov, colcov=colcov, size=size, random_state=rng
+        )
+
+    pymc_dist = pm.MatrixNormal
+
+    mu = np.random.random((3, 3))
+    row_cov = np.eye(3)
+    col_cov = np.eye(3)
+
+    pymc_dist_params = {"mu": mu, "rowcov": row_cov, "colcov": col_cov}
+    reference_dist_params = {"mu": mu, "rowcov": row_cov, "colcov": col_cov}
+    expected_rv_op_params = {"mu": mu, "rowcov": row_cov, "colcov": col_cov}
+
+    reference_dist = lambda self: functools.partial(
+        self.matrixnormal_rng_fn, rng=self.get_random_state()
+    )
+    sizes_to_check = [None, (1), (2, 3)]
+    sizes_expected = [(3,), (1, 3), (2, 3, 3)]
+
+    tests_to_run = [
+        "check_rv_size",
+        "check_pymc_params_match_rv_op",
+        "check_pymc_draws_match_reference",
```

Comment:
This test fails, strangely the first element matches everytime. @brandonwillard is this related to https://github.com/aesara-devs/aesara/issues/209 ? There seems to a similar operation involved here.